show/main.py

```diff
@@ -951,10 +980,9 @@ def aging_time(ctx):
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
 def route_map(route_map_name, verbose):
     """show route-map"""
-    cmd = 'sudo {} -c "show route-map'.format(constants.RVTYSH_COMMAND)
+    cmd = ['sudo', constants.RVTYSH_COMMAND, '-c', 'show route-map']
     if route_map_name is not None:
-        cmd += ' {}'.format(route_map_name)
-    cmd += '"'
+        cmd += [str(route_map_name)]
```

Comment:
>route_map_name [](http://example.com/codeflow?start=20&length=14)

It should be appended to embedded `show route-map` command.

Reply:
Could you make sure this is tested?

Reply:
fixed, thanks!

Reply:
To simplify:
`cmd[-1] += ' {}'.format(route_map_name)`